packages/tree/src/notebook-tree.ts

```diff
@@ -0,0 +1,16 @@
+import { TabBarSvg } from '@jupyterlab/ui-components';
+
+import { TabPanel } from '@lumino/widgets';
+
+import { INotebookTree } from './token';
+
+export class NotebookTreeWidget extends TabPanel implements INotebookTree {
```

Comment:
Let's add a docstring to the class and `constructor` for consistency with the rest of the code base?